src/Layer.js

```diff
@@ -138,7 +115,7 @@ export default class Layer {
     }
 
     /**
-     * Add this layer to a map.
+     * Add this layer to a map. It waits for the map to be loaded.
```

Comment:
I would change the wording

Reply:
`Wait for the map to load`?

Reply:
I would say something like: If the map's style is not loaded yet it will wait for it to add the layer as soon as possible